src/Uno.UI/DataBinding/BindingExpression.cs

```diff
@@ -347,13 +347,6 @@ private void ApplyExplicitSource()
 					this.Log().DebugFormat("Applying explicit source {0} on {1}", ExplicitSource?.GetType(), _view.Target?.GetType());
 				}
 
-				var resourceName = ExplicitSource as string;
```

Comment:
Could you provide the scenario when `ExplicitSource` is set ? Are you assigning a string directly to the `Binding.Source` ?

Reply:
Yes, and yes.  I have a markup extension that returns the string representation of a geometry for a path.  Here is an example:
                    <PathIcon Width="20"
                              Height="20"
                              Data="{Binding Source={icon:MaterialExtension Kind=Account}}"
                              Foreground="Yellow" />